### ISSUE TITLE
how get the value of the theano.tensor variable  

### ISSUE THREAD

**Issue by rahashwini** [Theano/Theano#6593]
My code is like this.
input_var = T.matrix(name='X_train')
target_var = T.ivector(name='y_train')

network = lasagne.layers.DenseLayer(network, len(y_train), nonlinearity=sigmoid, W=lasagne.init.GlorotNormal())
    params_original = lasagne.layers.get_all_param_values(network)
    params = lasagne.layers.get_all_params(network, trainable=True)
    layers = lasagne.layers.get_all_layers(network)
    get_params = lasagne.layers.get_all_param_values(network)

    #define loss function
    prediction = lasagne.layers.get_output(network)
    dim =  lasagne.layers.get_output_shape(network)
    loss = lasagne.objectives.categorical_crossentropy(prediction, target_var)

I want the value of prediction variable for further computation. I have tried to get it using T.eval(prediction). But I am getting the error with this option. Please suggest any other options.

**Comment by lamblin** [comment]
Please take the time to get familiar with Lasagne and Theano.
For instance, https://github.com/Lasagne/Lasagne/blob/master/docs/user/tutorial.rst, especially the "compilation" part.
